tox.ini

```diff
@@ -4,24 +4,18 @@ skip_missing_interpreters=True
 requires = pip
            virtualenv >= 20.2.2
 
-[testenv:py39]
-basepython=python3.9
-
 [testenv]
 deps=
     -rrequirements.dev.txt
     -rrequirements.txt
 extras=
     dataflow_graphic
-commands=py.test --log-level DEBUG --cov=inmanta --cov-report term --cov-report xml --junitxml=junit-{envname}.xml -vvv --durations=50 tests/
+# Set the environment variable EXTRA_ARGS='--fast' to run in fast mode
```

Comment:
I think we may have to document this in the readm.md and broadcast it on slack?

Reply:
Good idea. I will do that.